dsgrid/utils/files.py

```diff
@@ -72,6 +72,7 @@ def load_data(filename, **kwargs):
             data = mod.load(f_in)
         except Exception:
             logger.exception("Failed to load data from %s", filename)
+            breakpoint()
```

Comment:
Seems like this needs to be removed before merging?

Reply:
thanks